src/workloads/streams/CurrencyConversion.yml

```diff
@@ -0,0 +1,73 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/altas-streams"
+Description: |
+  Streaming pipeline that applies currency conversion from USD to EU.
+
+Keywords:
+- streams
+
+GlobalDefaults:
+  DatabaseName: &DatabaseName "test"
+  StreamProcessorName: &StreamProcessorName "sp"
+
+Actors:
+- Name: Setup
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Phase: 0
```

Comment:
Nit: Common practice is not to provide the phase numbers and instead infer them from the order of the list.

Reply:
ahh yeah i did this on purpose just so its much easier to parse exactly what each actor is doing in each phase, b/c i misaligned the nops earlier, so having the phase number explicitly there was easier for me to parse this at first glance personally